tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/IcuTextWriter.java

```diff
@@ -238,9 +238,9 @@ private static boolean mustBeArray(boolean topValues, String name, RbPath rbPath
     }
 
     private void printArray(RbValue rbValue, boolean quote, boolean isSequence, PrintWriter out) {
-        for (int n = 0; n < rbValue.size(); n++) {
+        for (String v : rbValue.getElements()) {
```

Comment:
easier for testing to allow the elements list to be accessed directly (it's immutable anyway).